src/views/managers.js

```diff
@@ -93,37 +117,45 @@ const Managers = () => {
       });
   };
 
-  return (
-    <div className="managers">
-      <div className="section-header">
-        <h2 className="page-title">Property Managers</h2>
-        <Link className="button is-rounded" to="/manage/managers">
-          + ADD NEW
-        </Link>
-      </div>
-      <div>
-        <Search placeholderMessage="Search property managers by name, property, or status"/>
-      </div>
-      <div className="invite-button">
-        <button className="button is-rounded" type="submit">
-          <FontAwesomeIcon
-            className="button__envelope-icon"
-            icon={faEnvelope}
-          />{" "}
-          Invite
-        </button>
-      </div>
-      <BootstrapTable
-        keyField="id"
-        data={PROPERTY_MANAGER_DATA}
-        columns={columns}
-        selectRow={selectRow}
-        bootstrap4={true}
-        headerClasses="table-header"
-        wrapperClasses="managers__table"
-      />
-    </div>
-  );
+  render() {
+    return (
+        <div className="managers">
+          <div className="section-header">
+            <h2 className="page-title">Property Managers</h2>
+            <Link className="button is-rounded" to="/manage/managers">
+              + ADD NEW
+            </Link>
+          </div>
+
+          <Search
```

Comment:
I can't seem to clear the filtering and see all results again once I remove my search term and hit "Enter", this might be a bug with the Search component that we should look into.

To Reproduce:
* Type "asdf" into the search bar and hit "Enter" (with the seeded Property Managers list)
* See no results shown
* Erase search term and hit "Enter"
* You'll still see no results